docs/source/deployment/databricks/databricks_notebooks_development_workflow.md

```diff
@@ -0,0 +1,293 @@
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/07_model_output', name='07_model_output', size=...),
+ FileInfo(path='dbfs:/FileStore/iris-databricks/data/08_reporting', name='08_reporting', size=...)]
+```
+
+After these cells have successfully run, you should comment the code inside them so their operations are not unnecessarily performed during notebook runs. The cells should appear as below:
+
+**Cell 1:**
+
+```ipython
+#dbutils.fs.cp(
+#    "file:///Workspace/Repos/<databricks_username>/iris-databricks/data",
+#    "dbfs:/FileStore/iris-databricks/data",
+#    recurse=True,
+#)
+```
+
+**Cell 2:**
+
+```ipython
+#dbutils.fs.ls("dbfs:/FileStore/iris-databricks/data")
+```
+
+### Run your project
+
+Create **four new cells** inside your notebook. You will fill these cells with code that runs your project. When copying the following code snippets, remember to replace `<databricks_username>` with your username on Databricks such that `project_root` correctly points to your project's location.
+
+1. Before you import and run your Python code, you'll need to install your project's dependencies on the cluster attached to your notebook. Your project has a `requirements.txt` file for this purpose. Add the following code to the first new cell to install the dependencies:
+
+```ipython
+%pip install -r "/Workspace/Repos/<databricks_username>/iris-databricks/src/requirements.txt"
+```
+
+2. To run your project in your notebook, you must load the Kedro IPython extension. Add the following code to the second new cell to load the IPython extension:
+
+```ipython
+%load_ext kedro.ipython
+```
+
+3. Loading the extension allows you to use the `%reload_kedro` line magic to load your Kedro project. Add the following code to the third new cell to load your Kedro project:
+
+```ipython
+%reload_kedro /Workspace/Repos/<databricks_username>/iris-databricks
+```
+
+4. Loading your Kedro project with the `%reload_kedro` line magic will define four global variables in your notebook: `context`, `session`, `catalog` and `pipelines`. You will use the `session` variable to run your project. Add the following code to the fourth new cell to run your Kedro project:
+
+```ipython
+session.run()
+```
+
+After completing these steps, your notebook should match the following image:
+
+![Databricks completed notebook](../../meta/images/databricks_notebooks_workflow_finished_notebook.png)
+
+Run the completed notebook using the `Run All` button in the top right of the UI:
+
+![Databricks notebook run all](../../meta/images/databricks_run_all.png)
+
+On the first run of your Kedro project, you will be prompted to consent to analytics, type `y` or `N` in the field that appears and press `Enter`:
+
+![Databricks notebook telemetry consent](../../meta/images/databricks_telemetry_consent.png)
+
+You should see logging output while the cell is running. After execution finishes, you should see output similar to the following:
```

Comment:
Is there a missing ` --env=databricks` in `%reload_kedro`, like in https://kedro--2628.org.readthedocs.build/en/2628/deployment/databricks/databricks_ide_development_workflow.html ?

Got an unexpected error when running the session:

```
DataSetError: Failed while loading data from data set SparkDataSet(file_format=csv, filepath=/Workspace/Repos/juan_luis_cano@mckinsey.com/iris-databricks-gh/data/01_raw/iris.csv, load_args={'header': True, 'inferSchema': True}, save_args={'header': True}).
[PATH_NOT_FOUND] Path does not exist: dbfs:/Workspace/Repos/juan_luis_cano@mckinsey.com/iris-databricks-gh/data/01_raw/iris.csv.
```

It's weird because the catalog definition looks like this:

```
example_iris_data:
  type: spark.SparkDataSet
  filepath: /dbfs/FileStore/iris-databricks/data/01_raw/iris.csv
  file_format: csv
  load_args:
    header: True
    inferSchema: True
  save_args:
    header: True
```

And I followed the commands at the beginning and got this correct `ls` output:

```
FileInfo(path='dbfs:/FileStore/iris-databricks-gh/data/01_raw/', name='01_raw/', size=0, modificationTime=1686208789000)
```

This error went away when adding `--env=databricks`.

Reply:
There's a chance I used the wrong starter all along (which would also explain my weird results in the logging PR). I'll start from scratch next week.

Reply:
No problem! Thanks for all your help. Let me know if we should jump on a call next week.

Reply:
Indeed I was using the wrong starter but I had named the directory like the new one, which confused me. I'll start the review from scratch.